.github/workflows/os-zoo.yml

```diff
@@ -166,3 +166,22 @@ jobs:
     - name: test
       working-directory: _build
       run: nmake test VERBOSE_FAILURE=yes HARNESS_JOBS=4
+  
+  self-hosted:
```

Comment:
All other jobs are per platform...  it's a little strange that this breaks that pattern.

Reply:
Otherwise, we would have duplicated code.

Reply:
I get that, and boy do we have that already!

It's just a little strange to see this lots of lines saying "alpine" or "linux" or "macos" or "windows"...  and then "self-hosted".

Reply:
IMO it is fine as is.